uxarray/grid/neighbors.py

```diff
@@ -268,3 +493,25 @@ def _prepare_xy_for_query(xy, use_radians):
         xy = np.deg2rad(xy)
 
     return xy
+
+
+def _prepare_cart_xyz_for_query(xyz):
```

Comment:
```suggestion
def _prepare_xyz_for_query(xyz):
```

this name should suffice